### ISSUE TITLE
Thumbnail generation does not start on my external drive.

### ISSUE THREAD

**Issue by kerbacho** [spacedriveapp/spacedrive#2757]
### Check for other issues

- [X] I've checked for other issues, and none exist.

### Issue description

It only generates the thumbnails when I open up the folders and subfolders manually.

### Platform and versions

- OS:
- Spacedrive:

<details><summary> Development tool versions (optional) </summary><pre>

</pre></details>

### Code of Conduct

- [X] I agree to follow the Code of Conduct.

**Comment by fogodev** [comment]
Are you exploring your folders through ephemeral indexer, or you added your external drive as a location?

**Comment by kerbacho** [comment]
I added the drive as location.

**Comment by kerbacho** [comment]
I'm on mac os 14.7 btw.

**Comment by fogodev** [comment]
Hmm, the media processor job ran without issues? Because when we add a location, we dispatch a job that fetches all your files and generate thumbnails for them eagerly.

**Comment by jamiepine** [comment]
Chiming in, I have this issue with external drives on Mac with 0.4.2. The thumbnail job instantly gets canceled every time without fail only on external drives. 

For me, the error is "too many files" which is a MacOS specific issue, if you run the command `ulimit -n 10240` in the Terminal app this issue will miraculously go away. It is a hotfix and we will figure something out soon.

**Comment by kerbacho** [comment]
Yes, it's not the first time that it happened. On another harddrive the thumbnails just appeared after a few days or so. The indexing went fine. But even the media view stays empty if I don't open a folder.Am 10.10.2024 22:22 schrieb fogodev ***@***.***>:
Hmm, the media processor job ran without issues? Because when we add a location, we dispatch a job that fetches all your files and generate thumbnails for them eagerly.

**Comment by kerbacho** [comment]
will try it out tomorrow, thanks!Am 10.10.2024 22:41 schrieb Jamie Pine ***@***.***>:
Chiming in, I have this issue with external drives on Mac with 0.4.2. The thumbnail job instantly gets canceled every time without fail only on external drives.
For me, the error is "too many files" which is a MacOS specific issue, if you run the command ulimit -n 10240 in the Terminal app this issue will miraculously go away. It is a hotfix and we will figure something out soon.

**Comment by jamiepine** [comment]
Closing due to v2 rewrite. If you encounter this issue in v2, please open a new issue.
